qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/customnavview/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -343,6 +347,16 @@ class MapboxNavigationViewCustomizedActivity : DrawerActivity() {
             viewModel.enableCompass,
             ::toggleEnableCompass
         )
+
+        MainScope().launch {
```

Comment:
You are not using lifecycleScope because it's needed in background mode with just the notification?

Reply:
it's for testing only